### PR TITLE
Feature sign up mail

### PR DESCRIPTION
**Pull request by BoAnd**
Still missing mail body. But please review.

### CODE REVIEW

File: stregsystem/models.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from stregsystem.templatetags.stregsystem_extras import money
 from stregsystem.utils import date_to_midnight
 from stregsystem.utils import send_payment_mail
+from stregsystem.utils import send_sign_mail
```
Comment (DelusionalLogic) [inline stregsystem/models.py]: The style is generally to do the python tuple import thing

File: stregsystem/models.py [hunk 2]
```diff
@@ -182,6 +183,10 @@ def __unicode__(self):
     def __str__(self):
         return active_str(self.active) + " " + self.username + ": " + self.firstname + " " + self.lastname + " | " + self.email + " (" + money(self.balance) + ")"
 
+    def save(self, *args, **kwargs):
```
Comment (DelusionalLogic) [inline stregsystem/models.py]: I could be wrong here, but won't this send the mail every time the balance of a user is saved?

Regardless, I'd prefer this be done in a modeladmin class. Placing things in the save makes it all very pasta.

File: stregsystem/models.py (hunk 1 above)
Comment (BoAnd) [reply stregsystem/models.py]: Like this: 
```
from stregsystem.utils import (
    date_to_midnight, 
    send_payment_mail, 
    send_sign_mail 
)
```

Comment (DelusionalLogic) [reply stregsystem/models.py]: Yeah

File: stregsystem/utils.py [hunk 3]
```diff
@@ -105,10 +99,24 @@ def send_payment_mail(member, amount):
     </html>
     """
 
-    msg.attach(MIMEText(html, 'html'))
+    send_mail(member.email, 'Stregsystem payment', html)
+
+def send_sign_mail(member):
+    html = f"""
```
[the remark this answers is not in the document]
Comment (BoAnd) [reply stregsystem/utils.py]: The first commented I made "Still missing mail body. But please review." :stuck_out_tongue_closed_eyes:

File: stregsystem/models.py (hunk 1 above)
Comment (BoAnd) [reply stregsystem/models.py]: 👍. I will fix it then.

File: stregsystem/utils.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Rene4100) [reply stregsystem/utils.py]: @BoAnd Where is the body of the mail?

[the remark this answers is not in the document]
Comment (BoAnd) [reply stregsystem/utils.py]: @Rene4100 It is in comit 839fb3a 👍

File: stregsystem/models.py [hunk 4]
```diff
@@ -187,8 +187,9 @@ def __str__(self):
         return active_str(self.active) + " " + self.username + ": " + self.firstname + " " + self.lastname + " | " + self.email + " (" + money(self.balance) + ")"
 
     def save(self, *args, **kwargs):
+        if (self.id == None):
```
[the remark this answers is not in the document]
Comment (BoAnd) [reply stregsystem/models.py]: This is now fixed. So ...

File: stregsystem/models.py (hunk 2 above)
Comment (BoAnd) [reply stregsystem/models.py]: Now it only sends the mail. When creating the user. So ...

File: stregsystem/utils.py [hunk 5]
```diff
@@ -111,10 +111,10 @@ def send_sign_mail(member):
             , the sports day (F-sportsdag), the cabin trip (Fyttetur) and so much <a href="http://www.fklub.dk/aktiviteter/start">more</a>!<br><br>
             The perhaps most important event is the Friday bread (FredagsFranskbrød)<br>
             each Wednesday at 10.00 o'clock in the cafeterie, where there will be 2 free<br>
-            slices of bread for each F-Klub member.<br><br>
+            slices of bread for each F-Klub member with butter.<br><br>
```
Comment (Rene4100) [inline stregsystem/utils.py]: "slices of bread with butter" instead

Comment (BoAnd) [reply stregsystem/utils.py]: Fixed.

Comment (fishbacon) [reply stregsystem/utils.py]: I really like the idea of buttering fembers.

**Comment by Rene4100** [comment]
Merge Merge Merge!

**Comment by hrjakobsen** [comment]
Implemented in #185

**Pull request closed without merging**